### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Shuyang-Liu** [TTdeveloping/text_classification#1]
Hello, sorry to bother you. I've cloned your codes and when I tried to run them, it stopped at "Using Wide Convolution" and then Process finished at exit code 0, I'd appreciate it a lot if you can possibly tell me what the bug is.

Following is the console info.
D:\Anaconda\python.exe "D:\Program Files\PyCharm Community Edition 2022.1.3\plugins\python-ce\helpers\pydev\pydevconsole.py" --mode=client --port=56139
import sys; print('Python %s on %s' % (sys.version, sys.platform))
sys.path.extend(['D:\\source\\NLP\\text_classification-master', 'D:\\source\\NLP\\text_classification-master\\Data', 'D:\\source\\NLP\\text_classification-master\\Dataload', 'D:\\source\\NLP\\text_classification-master\\DataUtils', 'D:\\source\\NLP\\text_classification-master\\model', 'D:\\source\\NLP\\text_classification-master\\Config', 'D:/source/NLP/text_classification-master'])
Python 3.9.7 (default, Sep 16 2021, 16:59:28) [MSC v.1916 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 7.29.0 -- An enhanced Interactive Python. Type '?' for help.
PyDev console: using IPython 7.29.0
Python 3.9.7 (default, Sep 16 2021, 16:59:28) [MSC v.1916 64 bit (AMD64)] on win32
runfile('D:/source/NLP/text_classification-master/main.py', wdir='D:/source/NLP/text_classification-master')
Process ID29380,Process Parent ID44000
Loaded config file sucessfully.
pretrained_embed : True
zeros : False
avg : False
uniform : False
nnembed : True
pretrained_embed_file : ./Data/embed/glove.sentiment.conj.pretrained.txt
train_file : ./Data/sst_binary/stsa.binary.trai
dev_file : ./Data/sst_binary/stsa.binary.dev
test_file : ./Data/sst_binary/stsa.binary.test
max_count : -1
min_freq : 1
shuffle : True
epochs_shuffle : True
save_pkl : True
pkl_directory : ./Save_pkl
pkl_data : pkl_data.pkl
pkl_alphabet : pkl_alphabet.pkl
pkl_iter : pkl_iter.pkl
pkl_embed : pkl_embed.pkl
save_dict : True
dict_directory : ./Save_dictionary
word_dict : dictionary_word.txt
label_dict : dictionary_label.txt
save_direction : ./Save_model
save_best_model_dir : ./Save_BModel
save_model : True
save_all_model : False
save_best_model : True
model_name : text_model
rm_model : True
wide_conv : True
model_cnn : True
model_bilstm : False
lstm_layers : 1
embed_dim : 300
embed_finetune : True
lstm_hiddens : 150
dropout_emb : 0.5
dropout : 0.5
conv_filter_sizes : 1,2,3,4
conv_filter_nums : 200
adam : True
sgd : False
learning_rate : 0.001
weight_decay : 1.0e-8
momentum : 0.0
clip_max_norm_use : True
clip_max_norm : 10
use_lr_decay : False
lr_rate_decay : 0.05
min_lrate : 0.000005
max_patience : 1
num_threads : 1
use_cuda : True
epochs : 500
early_max_patience : 30
backward_batch_size : 1
batch_size : 16
dev_batch_size : 16
test_batch_size : 16
log_interval : 100
***************************************
Data Process : True
Train model : True
Test model : False
t_model : None
t_data : None
predict : False
***************************************
use GPU training now:
starting init cuda seed 255
load data for process or pkl data
PROCESS DATA:
processing data............
Loading data:......
data path['./Data/sst_binary/stsa.binary.trai', './Data/sst_binary/stsa.binary.dev', './Data/sst_binary/stsa.binary.test']
load data form./Data/sst_binary/stsa.binary.trai
reading the 6920 line	shuffle train data......
load data form./Data/sst_binary/stsa.binary.dev
reading the 860 line	load data form./Data/sst_binary/stsa.binary.test
reading the 1820 line	train sentence 6920,dev sentence 872,test sentence 1821.
Start Build Vocab......
the length of train data 6920
the length of data that create Alphabet 6920 
*************   create 1 iterator    ***********  
the all data has been created iterator.
*************   create 2 iterator    ***********  
the all data has been created iterator.
*************   create 3 iterator    ***********  
the all data has been created iterator.
.
100%|██████████| 24049/24049 [00:00<00:00, 51248.36it/s]
loading pre_train embedding by nn.Embedding for out of vocabulary.
Words count 14830, Embed dim 300.
Exact count 14358 / 14830
Fuzzy count 0 / 14830
  INV count 14358 / 14830
  OOV count 472 / 14830
  OOV radio ===> 3.29%
****************************************
All Data/Alphabet/Iterator Use Time 5.385
***************************************
the learning algorithm is Adam.
embed_num : 14830,class_num : 2
PaddingID 1
word_dict_directory ：./Save_dictionary/dictionary_word.txt
label_dict_directory : ./Save_dictionary/dictionary_label.txt 
Saving dictionary.........
Save dictionary has been finished.........
Saving dictionary.........
Save dictionary has been finished.........
copy dictionaconry to ./Save_model\2022-10-04_14-59-33
********************************************************
Using Wide Convolution
Process finished with exit code 0
